app/Console/Commands/DeviceList.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Console\LnmsCommand;
+use App\Models\Application;
+use App\Models\Device;
+use App\Models\Ipv4Address;
+use App\Models\Ipv6Address;
+use App\Models\Port;
+use App\Models\Sensor;
+use App\Models\Storage;
+use Symfony\Component\Console\Input\InputOption;
+
+class DeviceList extends LnmsCommand
+{
+    protected $name = 'device:list';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+        $this->addOption('json', 'j', InputOption::VALUE_NONE);
+        $this->addOption('apps', 'a', InputOption::VALUE_NONE);
+        $this->addOption('ports', 'p', InputOption::VALUE_NONE);
+        $this->addOption('ip', 'i', InputOption::VALUE_NONE);
+        $this->addOption('storage', 's', InputOption::VALUE_NONE);
+        $this->addOption('sensors', 'S', InputOption::VALUE_NONE);
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle(): int
+    {
+        $devices = Device::where('device_id', '>', 0)->get();
```

Comment:
```suggestion
        $devices = Device::all();
```

Reply:
However, if you use my query below, this isn't needed here and should be moved down.